extensions/inpost-parcel-lockers/src/find-lockers.tsx

```diff
@@ -24,7 +27,12 @@ export default function Command() {
   );
 
   return (
-    <List isLoading={isLoading} onSearchTextChange={setSearchText} searchBarPlaceholder="Enter your postcode" throttle>
```

Comment:
I removed `throttle` because it allows the normalisation of the entered postcode to happen more immediately. It shouldn't have any negative effects in terms of creating more HTTP requests, as only strictly validated postcodes are ever sent to the API.